utils/index.ts

```diff
@@ -19,6 +19,7 @@ export const DERIVATION_PATH = {
 };
 
 const JSON_RPC_PATH = 'http://3.72.109.56:8545';
+const LOCAL_SERVER_PATH = 'http://localhost:3000'
```

Comment:
Can we use this in the multiple place in the code where we use this URI ? (screens/Wallet/Wallet.tsx, WalletSendConfirm.tsx, ...)